### PR TITLE
Add CS Fundamentals International and change category ordering

### PR DESCRIPTION
**Pull request by balderdash**
The PMs are getting very specific about their desired category order, so I think it's now easier to use the category hash to define the category order instead of adding 3 more overrides to `CATEGORY_ORDERING_PRIORITY`.

I added friendly names for `csf_international`, `express`, and `pre-express` to the i18n gsheet. I'll do a staging scoop immediately after merging this PR. Courses A-F already have friendly names.

Before:
image
image

After:
image
image

### CODE REVIEW

File: lib/cdo/script_constants.rb [hunk 1]
```diff
@@ -141,7 +143,11 @@ def self.position_in_category(script, category)
   end
 
   def self.category_priority(category)
-    CATEGORY_ORDERING_PRIORITY[category.to_sym] || 0
+    if category == OTHER_CATEGORY_NAME
+      1e6 # 'other' goes at the end
```
Comment (Bjvanminnen) [inline lib/cdo/script_constants.rb]: Could be something like `CATEGORIES.keys.length` which perhaps makes it a little clearer (and also protects us against the case where we literally have a million categories :-P)

**Comment by Bjvanminnen** [comment]
Note: You could presumably scoop strings before merging PR if your strings are just new ones.

**Comment by Bjvanminnen** [comment]
Might be helpful to include before/after screenshots of the dropdown to make it more clear exactly what's changing.

**Comment by Bjvanminnen** [comment]
We also have some stories/tests in apps that hard-code category_priority (and include -1). It would be nice to also update those to the new numbering system.

File: lib/cdo/script_constants.rb [hunk 2]
```diff
@@ -141,7 +143,11 @@ def self.position_in_category(script, category)
...
+    else
+      CATEGORIES.keys.find_index(category.to_sym)
+    end
```
Comment (Bjvanminnen) [inline lib/cdo/script_constants.rb]: I do think this approach in general makes ordering much easier to reason about :)

**Comment by balderdash** [comment]
I think I found all the tests that needed updates, and I added screenshots to the description. @Bjvanminnen PTAL

**Pull request merged**